jemalloc.spec

```diff
@@ -1,8 +1,8 @@
-### RPM external jemalloc 5.2.1
+### RPM external jemalloc 5.2.1-peakmem
 
 %define tag %{realversion}
-%define branch master
-%define github_user jemalloc
+%define branch 52x-peakmem
+%define github_user fwyzard
```

Comment:
If we go with the patch, we should probably fork `jemalloc` under https://github.com/cms-externals/

Reply:
Sure - this is just a way to show what the changes would be, and make sure it doesn't break anything in CMSSW.